### DISCUSSION TITLE
At Random times (quite frequently) adding new rows stops working

### DISCUSSION THREAD

**Discussion by davhad** [AntlerVC/firetable#1316] [Support / Q&A] [answered]
Here's the jam video...
https://jam.dev/c/47f7975e-ec96-433d-be60-f01dc784ba44
Refreshing the browser typically does the trick, but it just happens quite often.

**Comment by BeeBombshell** [comment, marked as answer]
Hey @davhad, are you adding your rows with a [random id](https://docs.rowy.io/how-to/add-row#with-random-id)? It's not very clear what's happening in the video, perhaps could you try to edit the fields from the side panel just to make sure the fields are getting saved?

**Reply by BeeBombshell** [reply]
Closing this discussion as inactive... If the issue still persists, feel free to create a new discussion.
